web_environment_ribbon/static/src/js/ribbon.js

```diff
@@ -1,35 +1,57 @@
-/******************************************************************************
-    Copyright (C) 2015 Akretion (http://www.akretion.com)
-    @author Sylvain Calador <sylvain.calador@akretion.com>
+/* Copyright 2015 Sylvain Calador <sylvain.calador@akretion.com>
+   Copyright 2015 Javi Melendez <javi.melendez@algios.com>
+   Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Affero General Public License as
-    published by the Free Software Foundation, either version 3 of the
-    License, or (at your option) any later version.
+(function ($) {
```

Comment:
I have tried and this is working as expected (changing colors through parameters), so maybe it's not so incompatible at all?